.github/sys-tests/testcase-2/test.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -e
+
+docker compose up --build -d --wait
```

Comment:
do we really need to build it every time?
is there some way to keep the docker image?